app/provider/api.py

```diff
@@ -7,12 +7,39 @@
 from django.shortcuts import get_object_or_404
 
 from .models import Provider
+from .schemas import ProviderListSchema
 from .schemas import ProviderSchema
 from .schemas import TranslationsSchema
 
 router = Router()
 
 
+def to_response(model: Provider, lang: LanguageCode) -> ProviderSchema:
```

Comment:
might name it `provider_to_response` for consistency

Reply:
Good point, will fix.